docker-compose.yml

```diff
@@ -73,16 +73,23 @@ services:
         image: rabbitmq:3.7-alpine
         ports:
             - "127.0.0.1:5672:5672"
+    ddagent5:
+        image: datadog/docker-dd-agent:latest
+        environment:
+            - DD_BIND_HOST=0.0.0.0
+            - DD_API_KEY=invalid_key_but_this_is_fine
+        ports:
+            - "127.0.0.1:8126:8126"
     ddagent:
-        image: datadog/agent-dev:gbbr-apm-build
+        image: datadog/agent:latest
```

Comment:
sounds like a good change but not directly related to this PR, would be great to have such changes in different PRs…

Reply:
Yep! Going to pull it out now.